lib/blocs/list_state.dart

```diff
@@ -1,5 +1,6 @@
 import 'package:equatable/equatable.dart';
 
+//TODO should this be split into two, one for up/down and one for just down
```

Comment:
Yeah I think so. Would be nice to have e.g. `TwoWayListState extends ListState` although I'm actually not sure whether extending is needed or desirable